lib/util.js

```diff
@@ -63,9 +64,57 @@ function sortLibraries(a, b) {
   return a.type.localeCompare(b.type);
 }
 
+// spinner constant
+let totalPackages = 1; // start from `1` as we have node policy tested all the time
+let spinner = 0;
+let processedCount = 0;
+let semVerLogged = false;
+function initSpinnerInfo(packagesCount, _spinner) {
```

Comment:
Initing here, implies there is an implicit object at play, could we make it an explicit object instead? As they convey their intent clearly and consistently.

Reply:
https://github.com/stefanpenner/supported/pull/8#discussion_r592557296